src/cljs_time/extend.cljs

```diff
@@ -23,20 +24,13 @@
   IEquiv
   (-equiv [o other]
     (and (instance? goog.date.Date other)
-         (.equals o other)))
+         (ct/equal? o other)))
```

Comment:
Ok, I see the point.

But I'd rather not change it for the sake of it. Could we leave this out, and revisit if it becomes an issue?